### PR TITLE
Lesson 4: Improvements

### PR DESCRIPTION
**Pull request by briangershon**
More clarification, including:

- helping user get initial project setup
- add more action sections explaining what to do
- create test file later in lesson and skip an initial placeholder test file

### CODE REVIEW

File: lessons/projects/4.mdx [hunk 1]
```diff
@@ -372,6 +365,12 @@ failure looks like:
 
 Here are tests we'll be adding:
 
+<ContentCallout emoji="💡" size="md" variant="info">
+In the next sections we'll be continuing to add groups of tests in new `describe` sections.
+Where do these go in the file? You have an existing section that starts with `describe('constructor', async () => {`.
+Ensure you paste the future `describe` sections immediately after where that previous section ends.
+</ContentCallout>
+
```
Comment (elPiablo) [inline lessons/projects/4.mdx:373]: comment out:: 
   // const symbol = await contract.symbol()
      expect(symbol).to.equal('Tier')
    })
  })//
  
new code here

//}) leave me here!! //

**Review by elPiablo — CHANGES_REQUESTED** [review]
Hey @briangershon 
I worked with John on the introduction part, and i added some other points. The pasting of the testing code is going to cause a problem with the syntax. YOu have a callout early on about the describe functionality, and where it stops and starts, unfortunately if you've never seen these code blocks before, you're simply not going to get it, and the best way to show it, is by actually modelling it. I would strongly suggest commenting out a bit of previous code when giving your new stuff. After MVP, we can come back later and actually use this as an opportunity to highlight different code blocks.

File: lessons/projects/4.mdx [hunk 2]
```diff
@@ -198,6 +167,11 @@ contract TierNFT is ERC721, Ownable {
 }
 ```
```
[the remark this answers is not in the document]
Comment (briangershon) [reply lessons/projects/4.mdx:169]: Good find!

File: lessons/projects/4.mdx [hunk 3]
```diff
@@ -198,6 +167,11 @@ contract TierNFT is ERC721, Ownable {
...
 
+## What do we want to test on our TierNFT contract?
+
+Now that the tools are set up, let's talk about what we want to test in our
+smart contract. We'll then add create tests.
```
[the remark this answers is not in the document]
Comment (briangershon) [reply lessons/projects/4.mdx:173]: You don't like "add" and "create" together? It's twice as powerful as just having one or the other!

(Thanks for catching!)

File: lessons/projects/4.mdx (hunk 1 above)
Comment (briangershon) [reply lessons/projects/4.mdx:430]: Ok, I rewrote this section with a preamble showing where the previous `describe` group of tests ends, and where the new one should go.

This is similar to what you suggested.

I also decided not to paste in the whole `tier-nft.test.js`. All the `describe` blocks are in the lesson.

File: lessons/projects/4.mdx [hunk 4]
```diff
@@ -372,6 +365,12 @@ failure looks like:
 
 Here are tests we'll be adding:
 
+<ContentCallout emoji="💡" size="md" variant="info">
+In the next sections we'll be continuing to add groups of tests in new `describe` sections.
+Where do these go in the file? You have an existing section that starts with `describe('constructor', async () => {`.
+Ensure you paste the future `describe` sections immediately after where that previous section ends.
```
[the remark this answers is not in the document]
Comment (briangershon) [reply lessons/projects/4.mdx]: True, that! ☀️

Added a bit of detail about the changes I made in previous comment.

**Comment by briangershon** [comment]
Thank you @elPiablo and @georgemac510 for your feedback -- I appreciate your attention to detail and focus on new coders!

**Pull request merged**